kubectl-plugin/pkg/util/test_utils.go

```diff
@@ -0,0 +1,40 @@
+package util
```

Comment:
I think it's better to name this file something like `kubeconfig.go`.

Reply:
renamed